WalletWasabi.Fluent/ViewModels/Wallets/Send/FeeChartViewModel.cs

```diff
@@ -253,7 +253,11 @@ public void UpdateFeeEstimates(Dictionary<int, int> feeEstimates, FeeRate? maxFe
 
 		SliderMinimum = 0;
 		SliderMaximum = confirmationTargetValues.Length - 1;
-		var confirmationTargetCandidate = ConfirmationTargetValues.MinBy(x => Math.Abs(x - Services.UiConfig.FeeTarget));
+
+		var confirmationTargetCandidate = CurrentConfirmationTarget < -1
+			? ConfirmationTargetValues.MinBy(x => Math.Abs(x - Services.UiConfig.FeeTarget))
+			: CurrentConfirmationTarget;
```

Comment:
This piece of logic is beyond my understanding 🙈

It suprised me that the candidate can actually be less than -1.

Reply:
It's a mistake, the default value is -1 so I should have checked 0 there.
(I only use less than condition due to fraction loss warning)

Fixed.